apps/vscode/src/package.json

```diff
@@ -82,6 +77,10 @@
         }
       ],
       "commandPalette": [
+        {
+          "command": "nx.generate.ui.fileexplorer",
+          "when": "false"
+        },
```

Comment:
Not related to this PR, but I hid this command from the command palette as it doesn't make sense there.